plugin/vim-gh-line.vim

```diff
@@ -48,14 +48,37 @@ if !exists('g:gh_cgit_url_pattern_sub')
     let g:gh_cgit_url_pattern_sub = []
 endif
 
-func! s:gh_line(action) range
+if !exists('g:gh_git_remote')
+    let g:gh_git_remote = ""
+endif
+
+func! s:gh_line(action, force_interactive) range
     " Get Line Number/s
     let lineNum = line('.')
     let fileName = resolve(expand('%:t'))
     let fileDir = resolve(expand("%:p:h"))
     let cdDir = "cd '" . fileDir . "'; "
 
-    let origin = system(cdDir . "git config --get remote.origin.url")
+    let l:remotes = system(cdDir . "git remote")
+    let l:remote_list = split(l:remotes, '\n')
+    if len(l:remote_list) == 0
+      echom "It seems the repo does not have any remote"
+      return
+    endif
+
+    " try to find git remote:
+    " if force interactive input, or g:gh_git_remote is not set, or
+    " g:gh_git_remote is not the remote of current file, try to find git
+    " remote name again
+    if a:force_interactive == 1 || g:gh_git_remote == "" || index(l:remote_list, g:gh_git_remote) < 0
```

Comment:
Would it be possible to add a new configuration variable, something like `g:gh_always_interactive` (off by default), so that the interactive menu is always popped up (when there's more than 1 remote) even if it has already popped up before?

Reply:
@KostyaEsmukov There are two new command added, and user can have their own key mapping to these command so that they will get the interactive input always.
```
command! GHIteractive call <SID>gh_line('blob', 1)
command! GBIteractive call <SID>gh_line('blame', 1)
```

Reply:
I see, but it would require to override the default `\gh` and `\gb` mappings, which seems to be less convenient than switching a config var.

Reply:
make sense!